net-mgmt/xymon-client/src/opnsense/mvc/app/views/Kumy/XymonClient/index.volt

```diff
@@ -0,0 +1,54 @@
+<script type="text/javascript">
+    $(document).ready(function() {
+        function spinStart(selector) {
+            $(selector).addClass("fa fa-spinner fa-pulse");
+        }
+        function spinStop(selector) {
+            $(selector).removeClass("fa fa-spinner fa-pulse");
+        }
+
+        mapDataToFormUI({"frm_Settings": "/api/xymonclient/settings/get"}).done(function(data) {
+            if ('frm_Settings' in data) {
+                updateServiceControlUI('xymonclient');
+            }
+        });
+
+        // Handle save button
+        $("body").on("click", ".save-button", function() {
```

Comment:
The `SimpleActionButton()` help reduce code here, see comment in `setActionHook()`